contracts/Controller.sol

```diff
@@ -614,13 +615,29 @@ contract Controller is Initializable, OwnableUpgradeSafe, ReentrancyGuardUpgrade
     /**
      * @notice get Otoken payout after expiry
      * @param _otoken Otoken address
-     * @param _amount amount of Otoken
-     * @return payout = cashValue * amount
+     * @param _amount amount of Otoken. Always in 1e18
+     * @return amount of collateral to payout
      */
     function _getPayout(address _otoken, uint256 _amount) internal view returns (uint256) {
         uint256 cashValue = calculator.getExpiredCashValue(_otoken);
+        uint256 amountInStrike = cashValue.mul(_amount).div(1e18);
 
-        return cashValue.mul(_amount).div(1e18);
+        OtokenInterface otoken = OtokenInterface(_otoken);
+        address strike = otoken.strikeAsset();
+        address collateral = otoken.collateralAsset();
+
+        uint256 payoutAmount;
+
+        if (strike == collateral) {
+            payoutAmount = amountInStrike;
```

Comment:
even this will fail the integration test right, because for calls the getExpiredCashValue is in USD, but the collateral is in ETH. So where does the USD to ETH price conversion happen?